src/commands/cmd_context.c

```diff
@@ -80,6 +82,10 @@ void CommandCtx_Free(CommandCtx *command_ctx) {
 		RedisModule_FreeThreadSafeContext(command_ctx->ctx);
 	}
 
+	// Report command to slowlog.
+	SlowLog_Add(command_ctx->command_name, command_ctx->graph_ctx->graph_name, command_ctx->query,
+				QueryCtx_GetExecutionTime());
```

Comment:
This call should be within the `if(command_ctx->query)` condition at 89. Bulk insert commands don't populate the `query` field, nor would we want to add them to the slow log.

Reply:
👍